hadoop-hdfs-project/hadoop-hdfs-rbf/src/test/java/org/apache/hadoop/hdfs/server/federation/router/TestRouter.java

```diff
@@ -238,12 +238,16 @@ public void testSwitchRouter() throws IOException {
   private void assertRouterHeartbeater(boolean expectedRouterHeartbeat,
       boolean expectedNNHeartbeat) throws IOException {
     final Router router = new Router();
-    Configuration baseCfg = new RouterConfigBuilder(conf).rpc().build();
```

Comment:
We should have both tests one with false and one without.

Reply:
Done